builtin/logical/pki/backend_test.go

```diff
@@ -5119,7 +5119,7 @@ var (
 	edCACert  string
 )
 
-func mountPKIEndpoint(t *testing.T, client *api.Client, path string) {
+func mountPKIEndpoint(t testing.TB, client *api.Client, path string) {
```

Comment:
Why did we switch all the t pointer params to value ones?

Reply:
Oh and please fix fmt errors.

Reply:
[`testing.TB` is the interface](https://pkg.go.dev/testing#TB) of things both [`testing.T` the struct](https://pkg.go.dev/testing#T) and [`testing.B` the struct](https://pkg.go.dev/testing#B) have. We directly pass `t` (the `*testing.T`) type into `mountPKIEndpoint` as it satisfies the `testing.TB` interface, so we don't change the _passed_ type, just the _accepting_ type to allow this function to be used by `testing.B` benchmark functions.

Reply:
Ah! that makes sense, sorry should have looked that up myself.

Reply:
Nah that's fine :-) When we were just testing, it made sense. But I had a `testing.B` and nothing wanted to work with the `*testing.T` type sigs until I found `testing.TB`. Then I swapped it out and hoped nothing broke... :D